pkg/comment_actions/parsing.go

```diff
@@ -29,15 +29,34 @@ type CommentArgs struct {
 	Rest    []string
 }
 
+func Contains(s []string, str string) bool {
+	existedOnce := false
+	for _, v := range s {
+		if v == str {
+			existedOnce = true
```

Comment:
If all we care about is the existence of the string, we might want to `break` the loop here and immediately return, ie 
```
func Contains(s []string, str string) bool {
	for _, v := range s {
		if v == str {
			return true
		}
	}

	return false
}
```

Reply:
@polyrain thank you for your feedback, I will removing all these and you can refer to https://github.com/zapier/tfbuddy/pull/38#discussion_r1316371589